db/schema.rb

```diff
@@ -10,14 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_07_27_095226) do
+ActiveRecord::Schema.define(version: 2020_07_28_145709) do
 
   # These are extensions that must be enabled in order to support this database
-  enable_extension "citext"
```

Comment:
Try to make sure you only commit the changes that your migration introduced (which should just be the version change and the create_table block)

Reply:
Will make sure to do this in the future, but have re-added the lines that were removed